_includes/v22.2/ui/active-transaction-executions.md

```diff
@@ -0,0 +1,46 @@
+## Active Executions table
+
+Click <img src="{{ 'images/common/ui-columns-button.png' | relative_url }}" alt="Column selector" /> to select the columns to display in the table.
+
+The Active Executions table gives details for each transaction fingerprint in the transaction:
+
+Column | Description
+-----|------------
+Transaction Execution ID | The execution ID of the transaction.
+Most Recent Statement | The most recently executed statement in the transaction.
+Status | The status of the execution: Preparing, Waiting, or Executing.
+Start Time (UTC) | The time the execution started.
+Elapsed Time | The time elapsed since the transaction started.
+Time Spent Waiting | The time the execution spent waiting and experiencing lock contention.
+Statements | The number of statements in the transaction.
+Retries | The number of times statements are retried.
+Application | The name specified by the [`application_name`](show-vars.html#supported-variables) session setting.
+
+To view details of an active transaction execution, click an execution ID in the **Transaction Execution ID** column to open the **Transaction Execution** details page.
+
+## Transaction Execution details page
+
+The details page provides the following details on the transaction execution.
+
+- **Start Time (UTC)**: the time the execution started.
+- **Elapsed Time**: the time elapsed since the transaction started.
+- **Status**: the status of the execution: Preparing, Waiting, or Executing.
+- **Priority**: the [priority](transactions.html#transaction-priorities) of the transaction.
+- **Full Scan**: whether the execution performed a full scan of the table.
+- **Internal Retries**: the number of retries of statements in the transaction.
+- **Last Retry Reason**: the reason for the last statement retry.
```

Comment:
Is there a small, limited number of possible reasons? If so, perhaps list them here, if they have succinct names?